deploy/02-annotated-nginx-deploy.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "1"
+    litmuschaos.io/chaos: "true"
+  creationTimestamp: "2020-03-30T23:07:17Z"
+  generation: 2
+  labels:
+    app: nginx
+  name: nginx
+  namespace: litmus
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        imagePullPolicy: Always
+        name: nginx
+        resources: {}
```

Comment:
@rahulchheda can we keep only the sample yaml spec w/o creationTimeStamp, resources, terminationMessagePath/Policy/SchedulerName etc.,

Reply:
Sure.